### DISCUSSION TITLE
Does uBO version of AG `hide-in-shadow-dom` scriptlet exist?

### DISCUSSION THREAD

**Discussion by piquark6046** [uBlockOrigin/uBlock-issues#2980] [Q&A] [answered]
To address https://github.com/AdguardTeam/AdguardFilters/issues/166984, i need a similar one.

**Comment by mapx-** [comment, marked as answer]
it seems working

`sbs.co.kr##sbs-player-ad-wrapper`

even it's not "seen" by the logger

**Reply by piquark6046** [reply]
Interesting. The filter does not work with a AdGuard CoreLib-based app. (But, on AG Browser Extension, it works.)

**Comment by garry-ut99** [comment]
Answering question in the title: it doesn't exist, however some discussion is taking place here :

- https://github.com/uBlockOrigin/uBlock-issues/issues/803
- https://github.com/uBlockOrigin/uBlock-issues/issues/2297
- https://github.com/uBlockOrigin/uBlock-issues/discussions/2280
- https://github.com/uBlockOrigin/uBlock-issues/issues/2971
